packages/openapi-spec/src/openapi-spec-v2.ts

```diff
@@ -240,27 +738,15 @@ export interface ParameterObject {
    */
   required?: boolean;
 
-  // TODO(bajtos) describe all properties,
-  // enforce polymorphic fields based on "in" value
-
   /**
    * _If `in` is any value other than "body":_
-   * The type of the parameter. Since the parameter is not located at
```

Comment:
This new comment does not make sense to me. It contains the condition ("If in is in any value other than body"), but does say what properties should be used when the condition is true. Maybe extend the comment with a reference to "ItemsObject"?

I also find the name "ItemsObject" as misleading, because typically it's not describing array items, but the top-level type like "string". I am proposing  something like `SimpleTypeSchema`, to relate to "full" `SchemaObject` type.

I makes me also wonder whether it's possible (and meaningful?) to create a shared type for schema properties that are allowed both in simple type definition and full JSON-schema-like definition?